esp8266/scripts/main.py

```diff
@@ -0,0 +1 @@
+# This script is run on boot
```

Comment:
Sounds good. I'd say we're not yet much ready for standalone running, but it makes sense to provide visible enough mechanism to do that for folks who want to try it.